exonum-java-proofs/src/main/java/com/exonum/binding/storage/proofs/map/flat/UncheckedFlatMapProof.java

```diff
@@ -22,8 +19,8 @@
 
   private final List<MapProofEntry> proofList;
 
-  UncheckedFlatMapProof(List<MapProofEntry> proofList) {
-    this.proofList = proofList;
+  UncheckedFlatMapProof(MapProofEntry[] proofList) {
+    this.proofList = Arrays.asList(proofList);
```

Comment:
@MakarovS any preconditions to check? E.g., that it is non-empty?

Reply:
Right now if we receive an empty `proofList` we consider it a correct one. Is it correct?

Reply:
Can’t say, got to see the spec or the reference implementation 🤷‍♂️